model_api/python/model_api/tilers/detection.py

```diff
@@ -123,8 +123,9 @@ def _merge_results(self, results, shape):
                 iou_threshold=self.iou_threshold,
             )
 
+        start_idx = 1 if self.tile_with_full_img else 0
```

Comment:
Why do we need these changes? It seems like removing the first tile in model_api/python/model_api/tilers/tiler.py is enough